infra/ansible/config/env.yaml

```diff
@@ -33,6 +33,7 @@ build_env:
     BAZEL_REMOTE_CACHE: 1
     SILO_NAME: "cache-silo-{{ arch }}-{{ accelerator }}-{{ clang_version }}"
     DISABLE_XRT: "{{ disable_xrt }}"
+    _GLIBCXX_USE_CXX11_ABI: 0
```

Comment:
Is this the right environment variable? We want to set whatever upstream expects, then we'll copy it here: https://github.com/pytorch/xla/blob/8412ca7f518ae33f5273c139e4ff5520979d0e87/setup.py#L213-L214

Reply:
Per https://github.com/pytorch/pytorch#install-pytorch, `_GLIBCXX_USE_CXX11_ABI` determines if we want to build pytorch with abi enabled.

It seems it works in r2.1 rc5.